tests/publiccloud/check_registercloudguest.pm

```diff
@@ -56,7 +56,7 @@ sub run {
             die('regionsrv-enabler-azure.timer is not enabled');
         }
 
-        if ($instance->ssh_script_output(cmd => 'sudo stat --printf="%s" /var/log/cloudregister') != 0) {
+        if ($instance->ssh_script_output(cmd => 'sudo stat --printf="%s" /var/log/cloudregister', proceed_on_failure => 1) != 0) {
```

Comment:
This is very nasty but it seems to work. What about the following? `test -s` returns 0 if the file exists and is nonempty. Like this we don't rely on the output of a command that results in an error.

```suggestion
        if ($instance->ssh_script_run(cmd => 'sudo test -s /var/log/cloudregister') == 0) {
```

Reply:
Sounds cool